From 8cd2abcbc90b8f77790a56c6a3ecf6a13369730a Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@cloudera.com>
Date: Thu, 21 Jun 2012 10:59:00 -0700
Subject: [PATCH 1193/1344] CDH-6250. Deb jdk package not compatible with hadoop java detection (at least for CDH3)

---
 bin/hadoop-config.sh |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/bin/hadoop-config.sh b/bin/hadoop-config.sh
index 96addf0..c6ad0e8 100644
--- a/bin/hadoop-config.sh
+++ b/bin/hadoop-config.sh
@@ -70,6 +70,8 @@ if [ -z "$JAVA_HOME" ]; then
     /usr/lib/jvm/java-6-sun \
     /usr/lib/jvm/java-1.6.0-sun-1.6.0.*/jre/ \
     /usr/lib/jvm/java-1.6.0-sun-1.6.0.* \
+    /usr/lib/jvm/j2sdk1.6-oracle \
+    /usr/lib/jvm/j2sdk1.6-oracle/jre \
     /usr/lib/j2sdk1.6-sun \
     /usr/java/jdk1.6* \
     /usr/java/jre1.6* \
-- 
1.7.0.4

